web/docs/formats/syslog.md

```diff
@@ -40,15 +40,25 @@ With this input, the parser will produce the following output, with the schema n
   "app_name": "evntslog",
   "process_id": "1370",
   "message_id": "ID47",
+  "structured_data": [
+    {
+      "id": "exampleSDID@32473",
+      "params": [
+        {
+          "key": "eventSource",
+          "value": "Application"
+        },
+        {
+          "key": "eventID",
+          "value": "1011"
+        }
+      ]
+    }
+  ],
   "message": "Event log entry"
 }
 ```
 
-:::note Structured Data
-The STRUCTURED-DATA field defined in RFC 5424, delimited by square brackets `[]` in the input,
-is currently not included in the resulting schema.
-:::
-
```

Comment:
The examples below need to be updated to include the `structured_data` field.

Reply:
`structured_data` is only applicable for RFC 5424 syslog messages, so there isn't going to be a field for them in 3164 messages.

Reply:
Makes sense. 🙈